lib/buildmodel.js

```diff
@@ -135,6 +135,66 @@ const BuildModel = class {
     stream(config, response) {
         return this.executor.stream(config, response);
     }
+
+    /**
+     * Get a build based on id
+     * @param  {Object}   config            Config object
+     * @param  {String}   config.table      The table name
```

Comment:
Why does the buldmdoel require a table parameter? Would the buildModel get every get a record from outside the builds table?